File: PandigitalMultiples/PM.java
/*
Project Euler # 38

*/

import java.util.Arrays;

public class PM {
    public static void main( String[] args ) {
        String ordered = "123456789";

        for ( int i=9123; i<=9876; i++ ) {
            String a = Integer.toString(i) + Integer.toString(2*i);
            char[] arr = a.toCharArray();
            Arrays.sort(arr);
            String sorted = new String(arr);
            if ( ordered.equals(sorted) )
                System.out.println(a);
        }

    }
}

File: PandigitalMultiples/PM.py
#!/usr/bin/env python3

"""
Project Euler # 38

"""

x = [str(i) for i in range(1,10)]

for i in range(9123, 9876):
    y = str(i) + str(2*i)
    if x == sorted(y):
        print(i, y)
